code/modules/projectiles/guns/ballistic/toy.dm

```diff
@@ -47,7 +47,7 @@
 	gun_flags = TOY_FIREARM_OVERLAY
 
 /obj/item/gun/ballistic/shotgun/toy/process_chamber(empty_chamber = 0)
-	..()
+	. = ..()
```

Comment:
All of the other procs do return ..() but this does . = ..(), why?

Reply:
they are more of base types, this one adds functionality at the end that relies on the parent